common.sh

```diff
@@ -141,6 +147,28 @@ function build_kb {
   go build $opts -o $tmp_root/kubebuilder/bin/kubebuilder ./cmd
 }
 
+function fetch_go_tools {
+  header_text "Checking for dep"
+  if ! is_installed dep; then
+    header_text "Installing dep"
+    go get -u github.com/golang/dep/cmd/dep
+  fi
+}
+
+function is_installed {
+  if [ command -v $1 &>/dev/null ]; then
+    return 0
+  fi
+  return 1
+}
+
+function fetch_git_tools {
+    header_text "Installing git lfs"
+    curl -s https://packagecloud.io/install/repositories/github/git-lfs/script.deb.sh | bash
```

Comment:
`curl | bash` ☹️ don't enjoy this

Reply:
let's bake it in to an image then

Reply:
yeah, I honestly didn't know where to put it but that was the plan. I'll make the image and we can figure out where to put it.

Reply:
Going to leave the git lfs artifact download here if that's okay. removing the install though.

Reply:
I could also make a separate script in `scripts/` which is basically:
```
CI=true # don't use sudo in scripts"
git lfs pull <repo>
,/install-test.sh
```

Reply:
(To invoke in prow)